test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -42,7 +45,7 @@
     public static final int ALTERNATIVE_CONTROLLER_PORT = 9093;
     public static final int ALTERNATIVE_REST_PORT = 9094;
     public static final TestExecutorFactory.TestExecutorType EXECUTOR_TYPE = TestExecutorFactory.getTestExecutionType();
-
+    public static final boolean SECURITY_ENABLED = isSecurityEnabled();
```

Comment:
`Security enabled` is an abstract term and today it means TLS and Auth are enabled, but tomorrow it might encompass other things. Please make it more precise here by breaking it down into two separate flags - one for auth and another for TLS. Since the system test framework currently only supports `auth`, you might use just the former.

Reply:
As per our earlier discussions and documentation, we plan to have only 1 environment for system tests with "security" enabled (for now this will be only auth enabled). Once I have tested with Auth +TLS enabled and it starts working fine, it will be auth + tls enabled pravega environment, hence the name isSecurityEnabled.
This variable controls running of all system tests with/without security.

Reply:
In this PR the support is for `auth` only, so as of now using a "security enabled" flag is misleading. Sure, I understand that the plan is to add support for TLS as well, but the code needs to represent what's in there today - not what it is supposed to to be tomorrow. A good way to tackle this might be to use separate flags but bring together using a method like isSecurityEnabled() for convenience.

Reply:
Done.